manifests/role/aws_mysqlexp.pp

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2020 The Regents of the University of Michigan.
+# All Rights Reserved. Licensed according to the terms of the Revised
+# BSD License. See LICENSE.txt for details.
+
+# Minimal aws with prometheus mysql exporter
+#
+# @example
+#   include nebula::role::aws
+class nebula::role::aws_mysqlexp {
```

Comment:
Since there's already an aws directory, I'd move this to aws/mysqlexp.pp and call it aws::mysqlexp, to be consistent with other aws roles.